client/stylesheets/_modal.scss

```diff
@@ -1136,7 +1136,45 @@ div[class="sp-replacer sp-light full-spectrum sp-active"] {
 /*~~ Join Button End ~~*/
 
 .modJoinWrapper {
+	width: 70% !important;
+	margin: 60px auto;
+}
+
+.modJoinWrapper h1 {
+	font-size: 2.4rem;
+	font-weight: 400;
+	margin-bottom: 0;
+}
+
+.modJoinWrapper ul li {
+	line-height: 2;
+}
+
+.modJoinOnly {
+	margin-top: 20px;
```

Comment:
Line should be indented with spaces, not tabs<br>Properties should be ordered font-size, margin-top